content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,255 @@
+security of data transfer.
+
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` uses an encryption protocol to encrypt
+communications. The protocol is called *Transport Layer Security* (TLS), although formerly it was
+known as *Secure Sockets Layer* (SSL).
+
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` occurs based upon the transmission of
+:abbr:`TLS (Transport Layer Security)`/:abbr:`SSL (Secure Sockets Layer)` certificates, which verify
+that a particular provider is who they say they are.
+
+.. note::
+   In this documentation and Odoo we will use the term "HTTPS certificate"
+   to define the fact that the SSL certificate is valid and allows
+   :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` connection.
+
+
+Why is it needed?
+=================
+
+In order to communicate with certain network devices (in particular payment terminals), the usage of
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` is mandatory. If the
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate isn't valid, some devices won't be
+able to interact with the :abbr:`IoT (Internet of Things)` Box.
+
+
+How to have obtain a Hypertext Transfer Protocol Secure (HTTPS) certificate
+===========================================================================
+
+The generation of the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate is automatic.
+
+The :abbr:`IoT (Internet of Things)` Box will send a specific request to `<https://www.odoo.com>`__
+that will send back the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate if the
+:abbr:`IoT (Internet of Things)` box and database are eligible.
+
+Internet of Things (IoT) eligibility
+------------------------------------
+
+ #. The database should be a **production** instance.
+    The database instance should not be a copy, a duplicate, a staging or a development environment.
+ #. The Odoo subscription must have an :guilabel:`IoT Box Subscription` line.
+    If you are unsure if there is an "IoT Box Subscription" line, contact your Account Manager or
+    Partner.
+
+Troubleshooting
+===============
+
+If anything goes wrong during the process of the "HTTPS certificate" generation or reception, a
+specific error will code be given on your :abbr:`IoT (Internet of Things)` box home page.
+
+
+ .. tip::
+    Accessing the :abbr:`IoT (Internet of Things)` box homepage will check for the presence of the
+    "HTTPS certificate" and will attempt its generation if missing.
+    As such, if you have an error on your :abbr:`IoT (Internet of Things)` box home page, you can
+    refresh the :abbr:`IoT (Internet of Things)` home page to see if the error disappears.
+
+    .. seealso::
+        - :ref:`iot/https-certificate`
+
+
+Hypertext Transfer Protocol Secure (HTTPS) certificate errors
+-------------------------------------------------------------
+
+`ERR_IOT_HTTPS_CHECK_NO_SERVER`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reason:
+    The configuration regarding the server is missing.
```

Comment:
it's called like this on the IoT box homepage :/ but I will add a reformulated version :)

Reply:
Maybe add a screenshot here? Or add to the description, or maybe an example block?

Reply:
Note that I mean this thing:
https://www.images.cybrosys.com/blog/Uploads/BlogImage/how-to-configure-iot-box-in-odoo-4.png

Personally for me it is explicit enough with the ..seealso link below. Let me know if you think it would be relevant to add a screenshot with the "configure" button highlighted.